client/my-sites/marketplace/components/billing-interval-switcher/index.jsx

```diff
@@ -0,0 +1,53 @@
+import { PlansIntervalToggle } from '@automattic/plans-grid/src';
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import SelectDropdown from 'calypso/components/select-dropdown';
+
+const Container = styled.div`
+	.plans-interval-toggle {
+		display: inline-flex;
+	}
+`;
+
+const PlansIntervalToggleLabel = styled.span`
+	font-size: 14px;
+	color: var( --studio-gray-60 );
+	margin-right: 10px;
+`;
+
+const BillingIntervalSwitcher = ( { billingPeriod, onChange, compact } ) => {
+	const translate = useTranslate();
+
+	if ( compact ) {
+		return (
+			<Container>
+				<SelectDropdown
+					selectedText={
+						billingPeriod === 'MONTHLY' ? translate( 'Monthly price' ) : translate( 'Annual price' )
+					}
+				>
+					<SelectDropdown.Item
+						selected={ billingPeriod === 'MONTHLY' }
+						onClick={ () => onChange( 'MONTHLY' ) }
+					>
+						{ translate( 'Monthly price' ) }
+					</SelectDropdown.Item>
+					<SelectDropdown.Item
+						selected={ billingPeriod === 'ANNUALLY' }
+						onClick={ () => onChange( 'ANNUALLY' ) }
+					>
```

Comment:
Do you think worth creating constants or an enum for those values(monthly, annually)? It can prevent errors from typos in the future and can also be used in other components.

Reply:
Done b9e4b7bad701c4675d2286684ce08034a6833044